README.md

```diff
@@ -23,6 +23,34 @@ GUI to load and analyze image data from electrical impedance tomography (EIT)
 
 The project setup is documented in [project_setup.md](project_setup.md). Feel free to remove this document (and/or the link to this document) if you don't need it.
 
+## Getting started
+
+### Installing Poetry
+
+EIT Dashboard makes use of [poetry](https://python-poetry.org/) to easily manage the needed packages. 
+`Poetry` can be installed both at system level following the [installation instructions](https://python-poetry.org/docs/#installation) (suggested) or by running:
+
+```console
+pip install poetry
+```
+in an environment with Python 3.10.
```

Comment:
show the recommended command here and potentially state that it's also possible via pip and give link for that.
`curl -sSL https://install.python-poetry.org | python3 -`